### ISSUE TITLE
Simple question about sourceMap

### ISSUE THREAD

**Issue by raphaelfabeni** [srod/node-minify#129]
Hey @srod ! First, congrats for the projects!
It's awesome. Let me ask you something: I'm trying to generate a `sourceMap` using _uglifyJS_ but I couldn't create the file.

```js
compressor.minify({
  compressor: 'uglifyjs',
  input: JSfiles,
  output: js_folder + '/build/main.min.js',
  sync: true,
  options: {
     outSourceMap: js_folder + '/build/main.min.js.map'
  },
  callback: function (err, min) {
    if(err) {
      console.log('Error to minify JS file → ', err);
     }
  }
});
```

I already changed the filenames (input, output and sourceMap) with absolute paths, but I got the minified file, but not the sourceMap. Is there something that I'm doing wrong?

Thanks!

**Comment by srod** [comment]
Hi, thank you !
I've checked UglifyJS docs and the map is returned as a string.
It's on my side, I will release a new version with the source map generated file.

**Comment by srod** [comment]
I've published 2.0.3

**Comment by raphaelfabeni** [comment]
Awesome man!
Thank you @srod !
🤘
